mlflow/store/tracking/rest_store.py

```diff
@@ -297,10 +297,38 @@ def get_experiment_by_name(self, experiment_name):
                 return None
             raise e
 
+    def _validate_new_param_value(self, run_id, new_params):
```

Comment:
I think we need to put the method in `AbstractStore`, so that FileStore, RestStore, SqlAlchemyStore can use it.